### PR TITLE
cluster: report paused info

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: pkg/spec/cluster.go [hunk 1]
```diff
@@ -20,7 +20,6 @@ import (
 	"time"
 
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/meta/metatypes"
```
Comment (hongchaodeng) [inline pkg/spec/cluster.go]: don't remove this?

**Comment by xiang90** [comment]
@hongchaodeng PTAL. tests fail due to known flaky.

File: pkg/spec/cluster.go [hunk 2]
```diff
@@ -163,6 +166,14 @@ func (cs *ClusterStatus) SetPhaseFailed() {
 	cs.Phase = ClusterPhaseFailed
 }
 
+func (cs *ClusterStatus) PausedControl() {
+	cs.ControlPaused = true
+}
+
+func (cs *ClusterStatus) Control() {
```
Comment (hongchaodeng) [inline pkg/spec/cluster.go]: ResumeControl()?

File: pkg/spec/cluster.go [hunk 3]
```diff
@@ -163,6 +166,14 @@ func (cs *ClusterStatus) SetPhaseFailed() {
 	cs.Phase = ClusterPhaseFailed
 }
 
+func (cs *ClusterStatus) PausedControl() {
```
Comment (hongchaodeng) [inline pkg/spec/cluster.go]: PauseControl()?

Comment (xiang90) [inline pkg/spec/cluster.go]: sure.

File: pkg/spec/cluster.go (hunk 2 above)
Comment (xiang90) [inline pkg/spec/cluster.go]: if you look at the code, we actually update the status in every round. we do not really care about the previous state of `paused`. Resume assumes the previous state is paused. So it is not suitable.

Comment (hongchaodeng) [inline pkg/spec/cluster.go]: ok

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**